src/js/ads.js

```diff
@@ -34,7 +34,7 @@ class VideoAds {
 				document.getElementsByTagName("head")[0].appendChild(googleSdkScript);
 			}
 
-			if (sdkScriptLoaded) {
+			if (sdkScriptLoaded || (window.google && window.google.ima)) {
```

Comment:
race condition - sometimes the event has already fired by the time you get to this line of script